### PR TITLE
Add task to disable checksum offload on default network interface

### PR DESCRIPTION
**Pull request by Rajalakshmi-Girish**
This is to fix https://jsw.ibm.com/browse/POWERCLOUD-16
Ref: https://calicousers.slack.com/archives/C0BCA117T/p1677501622349089

### CODE REVIEW

File: roles/install-k8s/tasks/main.yml [hunk 1]
```diff
@@ -1,3 +1,7 @@
+- name: Disable tx-checksumming on default network interface
```
Comment (mkumatag) [inline roles/install-k8s/tasks/main.yml:1]: lets add some why we are doing this also a follow up task if any to remove this in the future

**Review by mkumatag — CHANGES_REQUESTED** [review]

File: roles/install-k8s/tasks/main.yml (hunk 1 above)
Comment (mtarsel) [reply roles/install-k8s/tasks/main.yml:1]: yes i agree, lets wait until we understand why this is needed for session affinity test to pass

Comment (Rajalakshmi-Girish) [reply roles/install-k8s/tasks/main.yml:1]: To be precise, this is not needed for the feature of session affinity.
This is required for Nodeport service calls that cross the default network interface on ppc64le.

**Comment by Rajalakshmi-Girish** [comment]
@mkumatag Can you PTAL now?

**Review by mkumatag — APPROVED** [review]
/lgtm

**Comment by mkumatag** [comment]
@Rajalakshmi-Girish lets update the submodule as well,.

**Pull request merged**